tests/lib/state_test.py

```diff
@@ -138,3 +140,138 @@ def _extra_state(self):
                 {'name': 'eth1', 'state': 'up', 'type': 'unknown'}
             ]
         })
+
+
+def _get_test_ipv4_routes():
+    return [
+        {
+            Route.DESTINATION: '198.51.100.0/24',
+            Route.METRIC: 103,
+            Route.NEXT_HOP_INTERFACE: 'eth1',
+            Route.NEXT_HOP_ADDRESS: '192.0.2.1',
+            Route.TABLE_ID: 50
+        },
+        {
+            Route.DESTINATION: '203.0.113.0/24',
+            Route.METRIC: 103,
+            Route.NEXT_HOP_INTERFACE: 'eth1',
+            Route.NEXT_HOP_ADDRESS: '192.0.2.2',
+            Route.TABLE_ID: 51
+        }
+    ]
+
+
+def _get_test_ipv6_routes():
+    return [
+        {
+            Route.DESTINATION: '2001:db8:a::/64',
+            Route.METRIC: 103,
+            Route.NEXT_HOP_INTERFACE: 'eth1',
+            Route.NEXT_HOP_ADDRESS: '2001:db8:1::a',
+            Route.TABLE_ID: 50
+        },
+        {
+            Route.DESTINATION: '2001:db8:b::/64',
+            Route.METRIC: 103,
+            Route.NEXT_HOP_INTERFACE: 'eth1',
+            Route.NEXT_HOP_ADDRESS: '2001:db8:1::b',
+            Route.TABLE_ID: 51
+        }
+    ]
+
+
+class TestAssertRouteState(object):
+    parametrize_ip_ver_routes = pytest.mark.parametrize(
```

Comment:
I do not understand how this is relevant to the tested code. The tested code is generic, it does not care what IP version is used, it actually does not even care what is the content.

What is the minimum number of routes you need for these tests? 2? 3?

Reply:
Changed to function `_get_mixed_test_routes()` which returns 1 IPv4 route and 1 IPv6 route.